benchmark_runner/common/virtctl/virtctl.py

```diff
@@ -0,0 +1,85 @@
+
+import os
+from typeguard import typechecked
+
+from benchmark_runner.common.logger.logger_time_stamp import logger_time_stamp, logger
+from benchmark_runner.common.oc.oc import OC, VMStatus
+from benchmark_runner.main.environment_variables import environment_variables
+
+
+class Virtctl(OC):
+    """
+    This class run virtctl commands
+    """
+
+    def __init__(self):
+        super().__init__()
+
+    @typechecked
+    @logger_time_stamp
+    def save_vm_log(self, vm_name: str, output_filename: str = '', namespace: str = environment_variables.environment_variables_dict['namespace']):
+        """
+        This method save vm log in log_path
+        :param vm_name: vm name with uuid
+        :param output_filename:
+        :return:
+        """
+        if not output_filename:
+            output_filename = os.path.join(self._run_artifacts, vm_name)
+        self.run(cmd=f"virtctl console -n {namespace} {vm_name} > {output_filename}", background=True)
+
+    @typechecked
+    @logger_time_stamp
+    def stop_vm_async(self, vm_name: str, namespace: str = environment_variables.environment_variables_dict['namespace']):
+        """
+        The method stop vm
+        @param vm_name:
+        @param namespace:
+        @return:
```

Comment:
For everything here, either the return should be listed here or the underlying command should be run without a "return" before it.